src/main.cpp

```diff
@@ -116,54 +89,45 @@ int Main(vector<string> args)
 
 				// push input parameters on stack
 				lua_newtable(L);
-					lua_pushfield(L, "rule", rule.c_str());
-					lua_pushfield(L, "matched", sv.c_str(), sv.length());
-					lua_pushfield(L, "line", sv.line_info().first);
-					lua_pushfield(L, "column", sv.line_info().second);
-					lua_pushfield(L, "choice", sv.choice());
+					lua_pushfield(L, "rule",    rule.c_str());
```

Comment:
I'm not a big fan of this type of alignment: As be seen here and in the change to `peglib.h`, it leads to gratuitous changes on unrelated lines. Therefore, I would suggest to use only single spaces.

Reply:
`lua_pushfield(L, "rule", rule)` avoids re-measuring the string's length.